flink-runtime/src/main/java/org/apache/flink/runtime/state/heap/HeapMapState.java

```diff
@@ -91,6 +93,22 @@ public UV get(UK userKey) {
 		return userMap.get(userKey);
 	}
 
+	@Override
+	public Map<UK, UV> getAll(Collection<UK> keys) {
+
+		Map<UK, UV> userMap = stateTable.get(currentNamespace);
```

Comment:
I feel like by now we could really introduce a helper method `getUserMap() {stateTable.get(currentNamespace);}`.